config/blacklight.yml

```diff
@@ -1,6 +1,6 @@
 development:
   adapter: solr
-  url: <%= ENV['SOLR_URL'] || "http://127.0.0.1:#{ENV.fetch('SOLR_DEVELOPMENT_PORT', 8983)}/solr/hydra-development" %>
+  url: <%= ENV['SOLR_URL'] %>
```

Comment:
Where does the environment variable `SOLR_URL` get set?

Reply:
All of the env variables are set by cloudformation template. All of these should be available both in Rails and in Bash.